src/include/migraphx/config.hpp

```diff
@@ -23,12 +23,23 @@
  */
 #ifndef MIGRAPHX_GUARD_CONFIG_HPP
 #define MIGRAPHX_GUARD_CONFIG_HPP
-
+#include <migraphx/version.h>
```

Comment:
This should only be included when `BUILD_DEV` is not defined.

Reply:
Done